pkg/apiserver/server.go

```diff
@@ -88,6 +88,10 @@ func Start() {
 	// Find a home snapshot routes
 	r.Path("/api/v1/snapshot/{backup}/logs").Methods("OPTIONS", "GET").HandlerFunc(handlers.DownloadSnapshotLogs)
 
+	// redactor routes
+	r.Path("/api/v1/redact/set").Methods("OPTIONS", "PUT").HandlerFunc(handlers.UpdateRedact)
```

Comment:
I don't think these actually need `set` and `get`.  That's what HTTP methods are for.